src/components/Editor/Resources/ResourceSeatingCapacity.vue

```diff
@@ -22,34 +22,36 @@
 
 <template>
 	<div class="resource-capacity">
-		<div class="resource-capacity__input">
-			<input type="number"
-				min="0"
-				:placeholder="placeholder"
-				:value="value"
-				@input.prevent.stop="changeValue">
-		</div>
+		<NcTextField :label="placeholder"
+			type="number"
+			min="0"
+			:placeholder="placeholder"
+			:value.sync="capacityValue" />
 	</div>
 </template>
 
 <script>
+import NcTextField from '@nextcloud/vue/dist/Components/NcTextField.js'
 export default {
 	name: 'ResourceSeatingCapacity',
+	components: {
+		NcTextField,
+	},
 	props: {
 		value: {
 			type: Number,
 			required: true,
 		},
 	},
+	data() {
+		return {
+			capacityValue: this.value,
+		}
+	},
 	computed: {
 		placeholder() {
 			return this.$t('calendar', 'Minimum seating capacity')
 		},
 	},
-	methods: {
-		changeValue(event) {
-			this.$emit('update:value', parseInt(event.target.value))
-		},
```

Comment:
This event still needs to be emitted as it is listened to one level above in `ResourceListSearch`.

Reply:
Done